wadas/domain/configuration.py

```diff
@@ -55,7 +56,11 @@ def check_version_compatibility(config_file_version):
     if wadas_version < config_file_version:
         return False
     if wadas_version > config_file_version:
-        # Add here conversion logic for older versions to support (if any)
+        # Example of compatibility check to list here:

```

Comment:
Maybe in this case we can think about extracting as much configuration data (which is still useful) possible.
But it'll be a pain to keep this behaviour when multiple config versions will be introduced.